geocruncher/GeologicalModel3D.py

```diff
@@ -5,100 +5,18 @@
 #
 
 from collections import namedtuple
-import json
 import numpy as np
-import pypotential3D as pypotential
-from gmlib import topography_reader
+import yaml
```

Comment:
@poluxxs This looks like a copy from gmlib? Can you please add a comment where you copied this from (path in gmlib and exact gmlib git version/hash), and document everything you changed compared to the gmlib original?

Reply:
I do not understand the question. "where you copied this from" ?

I used only either code from gmlib or code that I wrote by myself. Moreover Almost everything that is pushed here is in addition to Gmlib since you asked me to only push changes instead of pushing whole functional files.

Reply:
Yes, but can you please add a comment where *exactly* you copied this from (original filename + gmlib version), and document all changes you have made compared to the gmlib version. You only need to do this for the GeologicalModel3D.py file, as only that seems to contain a lot of copied code.

Reply:
Or to put it bluntly, please document what you are doing. The next person working on Geocruncher will not know that this code is copied from gmlib. So this fact *needs* to be documented.

Reply:
So just write a readme or directly in the code?
I can also just directly push everything here so with one pull and one pip install eveything work.

I guess you prefer a readme^

Reply:
We are only talking about the file GeologicalModel3D.py. Put a comment at the top of that file, stating where you copied it from. Add a comment on top of the functions that you have changed in the file GeologicalModel3D.py explaining what you changed and why.

Reply:
Thanks for adding the comments.